composables/timeline.ts

```diff
@@ -14,7 +14,34 @@ function removeFilteredItems(items: mastodon.v1.Status[], context: mastodon.v1.F
   const isFiltered = (item: mastodon.v1.Status) => (item.account.id === currentUser.value?.account.id) || !item.filtered?.find(isStrict)
   const isReblogFiltered = (item: mastodon.v1.Status) => !item.reblog?.filtered?.find(isStrict)
 
-  return [...items].filter(isFiltered).filter(isReblogFiltered)
+  const homeFilter = useHomeFilter()
+  const { bot, sensitive, repost, mutual, tag } = homeFilter.value
+
+  const isClientSideHomeFiltered = (item: mastodon.v1.Status) => {
```

Comment:
1. Somehow the filter needs to be re-applied when the filter changes
2. Not sure right now if the filter is also applied to the "stream" (but for now this is not my prio...)